style.css

```diff
@@ -0,0 +1,58 @@
+* {
```

Comment:
why css, not sass?

Reply:
beacuse is too small project

Reply:
it's not too small project ;) you should train your skills in sass :)